src/ArborX_LinearBVH.hpp

```diff
@@ -106,6 +120,12 @@ class BoundingVolumeHierarchy
     return Kokkos::subview(_internal_and_leaf_nodes,
                            std::make_pair(size() - 1, 2 * size() - 1));
   }
+  Kokkos::View<node_type *, MemorySpace> getLeafNodes() const
```

Comment:
Any reason not to const the value type?   `Kokkos::View<node_type const *, ...>`